src/apps/statements/service/storeStatements/queriables/getMetadataFromStatement/getNumericQuestionMetadata.ts

```diff
@@ -0,0 +1,24 @@
+import { get, has } from 'lodash';
+
+import ActivityInteractionType from '../../../../models/ActivityInteractionType';
+import Statement from '../../../../models/Statement';
+
+export const getNumericQuestionMetadata = (statement: Statement)
+  : {readonly [key: string]: any} => {
+  if (
+    !(
+      get(
+        statement.object,
+        ['definition', 'interactionType'],
+      ) === ActivityInteractionType.NUMERIC
+      && has(statement, ['result', 'response'])
+    )
+  ) {
+    return {};
+  }
+
+  const numericQuestionString = get(statement, ['result', 'response']);
+  const numericQuestion = parseFloat(numericQuestionString);
```

Comment:
What about `min` and `max` according to [this](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Data.md#response-patterns)?

> **numeric**
> 
> A range of numbers represented by a minimum and a maximum delimited by [:]. Where the range does not have a maximum or does not have a minimum, that number is omitted but the delimiter is still used. E.g. [:]4 indicates a maximum for 4 and no minimum. Where the correct response or learner's response is a single number rather than a range, the single number with no delimiter MAY be used.

@ryansmith94 , should the response be like it is below?

```
{ "https://learninglocker.net/numeric-response": {min: <NUMBER>, max: <NUBMER>}}
```

Reply:
Ah wow good spot Vadim, hadn't actually noticed that the response to a numeric interaction could be a range rather than a single number, I've never seen it used as a range in practice. That makes things more interesting.

I propose that when the `result.response` contains `[:]` we actually use a different metadata property `https://learninglocker.net/numeric-range-response` with the two keys min and max in an object as you've suggested Vadim. Otherwise, if `result.response` does not contain `[:]` then we should use this existing metadata property `https://learninglocker.net/numeric-response` and store the parsed float number. I've put some examples below to help illustrate this proposal, let me know what you both think.

### Numeric Range Example
`result.response` = `"1[:]10"`

#### Expected Metadata
```json
{ "https://learninglocker.net/numeric-range-response": { "min": 1, "max": 10 }}
```

### Numeric Example
`result.response` = `"10"`

#### Expected Metadata
```json
{ "https://learninglocker.net/numeric-response": 10}
```